Podfile.lock

```diff
@@ -63,4 +63,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 7d7bea635e8c87f81a678520b81e3ce20b2f4e86
 
-COCOAPODS: 1.2.0
+COCOAPODS: 1.2.1
```

Comment:
We could probably update to CocoaPods 1.3.1, as well as update the external dependencies to the most recent versions.

Reply:
👍 There is always this problem with different versions of cocoapods between developers. I suggest using `bundler` and `Gemfile` to specify which version of Cocoapods project uses. It has one drawback - each `pod` command has to be prefixed with `bundle exec`, e.g. `bundle exec pod install` but this will ensure that correct version of cocoapods is used.

Reply:
It seems like a good idea